.github/workflows/build.yaml

```diff
@@ -0,0 +1,33 @@
+name: "Build and push docker container"
+
+on:
+  pull_request:
+  workflow_dispatch:
+    
+jobs:
+  cluster-bootstrap-controller:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Install Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.17.x
+    - name: Checkout code
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+        ref: ${{ github.event.pull_request.head.sha }}
+    - name: Build docker image
+      run: |
+        make docker-build
```

Comment:
We _might_ have to `git fetch tags --force` to get the right tag when releasing but lets test w/out